tiler/pyproject.toml

```diff
@@ -0,0 +1,14 @@
+name = "LandGriffon Tiler"
+version = "0.1.0"
+
+[tool.pytest.ini_options]
+pythonpath = [
+  "."
+]
+env = ["REQUIRE_AUTH=true",
+    "API_HOST=",
+    "API_PORT=3000",
+    "S3_BUCKET_NAME=mybucket",
+    "ROOT_PATH=",
+    "TITILER_PREFIX=/cog",
+    "TITILER_ROUTER_PREFIX=/cog"]
```

Comment:
I explored different options for setting test env vars in python, I decided to add this, even tho we can inject them, I think we should provide a way to have some test env vars as we do in a node config package, instead of adding a env.test file

Reply:
hmm don't know, never seen anything like this before for testing :shrug:. Up to you!